ui_framework/src/components/form/select.js

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+
+const SELECT_SIZE = [
+  'small',
+  'medium',
+  'large',
+];
+
+const KuiSelect = ({
+  className,
+  onChange,
+  value,
+  isInvalid,
+  isDisabled,
+  size,
+  children,
+  ...rest
+}) => {
+  const classes = classNames('kuiSelect', className,
+    { 'kuiSelect-isInvalid': isInvalid },
+    { 'kuiSelect--small': size === 'small' },
+    { 'kuiSelect--large': size === 'large' },
```

Comment:
This can be written as one object:

```js
const classes = classNames('kuiSelect', className, {
  'kuiSelect-isInvalid': isInvalid,
  'kuiSelect--small': size === 'small',
  'kuiSelect--large': size === 'large',
});
```

Reply:
Cool syntax! Ok.